app/models/course_user.rb

```diff
@@ -101,6 +101,13 @@ def self.user?(user)
     with_approved_state.exists?(user: user)
   end
 
+  # Test whether this course_user is a manager (i.e. manager or owner)
+  #
+  # @return [Boolean] True if course_user is a staff
+  def manager?
```

Comment:
This overrides the default AR magic method `manager`, I would prefer to name it `manager_or_owner?`